cmd/minishift/cmd/profile/profile_list.go

```diff
@@ -44,7 +45,15 @@ func displayProfiles(profiles []string) {
 	for _, profile := range profiles {
 		vmStatus := cmdUtil.GetVMStatus(profile)
 		if profile == activeProfile {
-			fmt.Fprintln(display, fmt.Sprintf("- %s\t%s\t(active)", profile, vmStatus))
+			fmt.Fprintln(display, fmt.Sprintf("- %s\t%s\t%s", profile, vmStatus, "(Active)"))
+		}
+	}
+
+	sort.Strings(profiles)
+	for _, profile := range profiles {
+		vmStatus := cmdUtil.GetVMStatus(profile)
```

Comment:
-1  two for loops.  We need to add more tab space before `(active)` and sort the profiles . Not sure if we want to add active profile as the first one https://github.com/minishift/minishift/issues/1592#issuecomment-338601379
In case you want to print the active profile first, you can just print the VM status of active profile (of the profile returned by profileActions.GetActiveProfile()) then in the for loop do not print anything for the active profile.

Reply:
@LalatenduMohanty Thanks, i also don't like the two loops, was thinking of how to avoid that.